codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/customizations/AdditionalErrorsDecorator.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package software.amazon.smithy.rust.codegen.server.smithy.customizations
+
+import software.amazon.smithy.model.Model
+import software.amazon.smithy.model.shapes.MemberShape
+import software.amazon.smithy.model.shapes.OperationShape
+import software.amazon.smithy.model.shapes.ServiceShape
+import software.amazon.smithy.model.shapes.StructureShape
+import software.amazon.smithy.model.traits.ErrorTrait
+import software.amazon.smithy.model.traits.RequiredTrait
+import software.amazon.smithy.model.transform.ModelTransformer
+import software.amazon.smithy.rust.codegen.smithy.customize.RustCodegenDecorator
+
+/**
+ * Add at least one error to all operations in the model.
+ *
+ * When this decorator is applied, even operations that do not have a Smithy error attatched,
+ * will return `Result<OperationOutput, OperationError>`.
+ *
+ * To enable this decorator write its class name to a resource file like this:
```

Comment:
```suggestion
/**
 * Add at least one error to all operations in the model.
 *
 * When this decorator is applied, operations that do not have a Smithy error attatched,
 * will return `Result<OperationOutput, InternalServerError>`.
 *
 * To enable this decorator write its class name to a resource file like this:
```

Reply:
I changed it to OperationError because InternalServerError is not really the error type, every operation gets its own enum and then the result is actually returning `Result<OperationOutput, OperationError>`

For example in my error.rs:

```
/// Error type for the `PutObject` operation.
/// Each variant represents an error that can occur for the `PutObject` operation.
#[derive(std::fmt::Debug)]
pub enum PutObjectError {
    #[allow(missing_docs)] // documentation missing in model
    InternalServerError(crate::error::InternalServerError),
}
```

and in operation_handler.rs:
```
#[async_trait::async_trait]
impl<B, Fun, Fut>
    crate::server_operation_handler_trait::Handler<B, (), crate::input::PutObjectInput> for Fun
where
    Fun: FnOnce(crate::input::PutObjectInput) -> Fut + Clone + Send + 'static,
    Fut: std::future::Future<
            Output = Result<crate::output::PutObjectOutput, crate::error::PutObjectError>,
        > + Send,
```

Reply:
Ok, works for me :)